### PR TITLE
Added #823: Added a swatch to identify the current character, and add…

### PR DESCRIPTION
**Pull request by SunburnedGoose**
…ed a property, custom, to the store.

Added a swatch to the character box for the current user. The store has a new property, current, that can be used to identify the current store. I've seen areas in the app where that code was used, but custom made. We can replace that code with the property.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Next
 
+* [#823](https://github.com/DestinyItemManager/DIM/issues/823) Added 'current' property to stores.
```
Comment (bhollis) [inline CHANGELOG.md]: I've been avoiding including "dev-only" stuff in the changelog - do you want to start?

**Comment by bhollis** [comment]
Cool - what does it look like?

File: CHANGELOG.md (hunk 1 above)
Comment (SunburnedGoose) [inline CHANGELOG.md]: It was one of those updates that seemed useful enough to want to elevate the awareness.

**Comment by SunburnedGoose** [comment]
Golden (err yellow?) Triangle on the character card.

screen shot 2016-09-04 at 7 29 33 pm

**Comment by kyleshay** [comment]
new size 

screen shot 2016-09-06 at 11 54 23 am

File: .eslintrc [hunk 2]
```diff
@@ -20,6 +20,8 @@
     "sum": true,
     "count": true,
     "gapi": true,
+    "Set": true,
+    "ga": true,
```
Comment (kyleshay) [inline .eslintrc]: oops- maybe we don't need this in here now?

Comment (SunburnedGoose) [inline .eslintrc]: Next time we update permissions, we're going to switch to the new permissions to use the latest 'ga'.

Comment (kyleshay) [inline .eslintrc]: cool. can clean those up then. we good to merge? thoughts on the gold wedge size @bhollis ?

**Comment by bhollis** [comment]
The new size is good!

File: .eslintrc [hunk 3]
```diff
@@ -20,6 +20,8 @@
     "sum": true,
     "count": true,
     "gapi": true,
+    "Set": true,
```
Comment (bhollis) [inline .eslintrc]: This looks like a bad merge - "Set" is already in the list.

**Pull request merged**